src/components/VoteScreen.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import CandidateStatus from './CandidateStatus.js';
+
+export default function VoteScreen() {
+  const [candidates, setCandidates] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get(
+          'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
+        );
+        setCandidates(response.data);
+      } catch (e) {
+        console.log(e);
+      }
+    };
+    fetchData();
+  }, [candidates]);
+
+  if (!candidates) {
+    return null;
+  }
+
+  candidates.sort((a, b) => b.voteCount - a.voteCount);
+
+  return (
+    <Wrapper>
+      <Intro>CEOS 프론트앤드 13기 개발팀장을 뽑자 🎉</Intro>
+      {candidates.map((candidate, index) => {
```

Comment:
```suggestion
  return (
    <Wrapper>
      <Intro>CEOS 프론트앤드 13기 개발팀장을 뽑자 🎉</Intro>
      {candidates 
      .sort((a, b) => b.voteCount - a.voteCount)
      .map((candidate, index) .map((candidate, index) => {
```

이렇게 해주는게 좀 더 가독성이 좋을 것 같아요 ! [메소드 체이닝 패턴 참고](https://webclub.tistory.com/528)